llvm/test/Feature/Repo/repo_pruning_GV_with_zeroinitializer.ll

```diff
@@ -0,0 +1,36 @@
+; This testcase tests that hash generation for GV with zeroinitializer.
+;
+; If a global variable has zeroinitializer during the RepoMetadataGenerationPass and
+; its initial value changes after the RepoMetadataGenerationPass, the digest of GV
+; needs to be recalculated.
+
+; The testcase includes three steps:
+; Step 1: Build the test Inputs/repo_GV_with_zeroinitializer.ll and create the database 'clang.db' which contains the Tickets.
+; Step 2: Build this file and check the function '-b' doesn't be pruned.
```

Comment:
"doesn't be pruned" -> "hasn't been pruned"?

Reply:
Fixed, thanks.